app/controllers/amusement_parks_controller.rb

```diff
@@ -0,0 +1,11 @@
+class AmusementParksController < ApplicationController
+    def show
+        @park = AmusementPark.find(params[:id])
+    end
+
+    private
+
+    def mechanic_params
+        params.permit(:id, :name, :admission_cost, :created_at, :updated_at)
```

Comment:
You typically only add strong params when you're grabbing data passed in via a form/text field, so it's not strictly necessary here